async_customerio/request_validator.py

```diff
@@ -0,0 +1,18 @@
+import hashlib
+import hmac
+
+
+def validate_signature(signing_key: str, timestamp: int, request_body: bytes, signature: str) -> bool:
+    """Validate that request was sent from Customer.io
+    Doc: https://customer.io/docs/journeys/webhooks/#securely-verify-requests
+
+    :param timestamp: unix timestamp, value from header  X-CIO-Timestamp
+    :param request_body: body of the request
+    :param signature: value from header X-CIO-Signature
+    :param token: value for SIGNING KEY from Customer.io
```

Comment:
The list of params is different than the function signature, could you please fix it?